channels/ir.m3u

```diff
@@ -17,6 +17,10 @@ https://live.al-zahratv.com/live/playlist2/index.m3u8
 https://nl.live21.ir/hls2/AMG_mid.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Persian" tvg-logo="https://i.imgur.com/UFV7YVy.jpg" group-title="",Andisheh TV
 http://livestream.5centscdn.com/cls090117/49ead3a7c6432c5358a086d078110c16.sdp/index.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Kurdish" tvg-logo="https://avaentertainment.net/img/logos/m-logo.png" group-title="Entertainment",AVA TV
+https://wowzaprod219-i.akamaihd.net/hls/live/1003949/c9aa559e/c9aa559e_1_2160000/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Azerbaijani" tvg-logo="http://azstartv.com/img/logo.png" group-title="Entertainment",AZ Star TV
+http://live.livestreamtv.ca/azstar/smil:azstar.smil/chunklist_w1956442391_b1000000.m3u8
```

Comment:
As I understand it, "Azstar TV" is being broadcast from Canada (https://azstartv.com/kunye.php), not Iran.

Reply:
A lot of channels targeting Iranians are not broadcasted from Iran. This channel is targeting Iranians who speak Azerbaijani.